### ISSUE TITLE
TypeError: Cannot read property 'title' of undefined

### ISSUE THREAD

**Issue by iFloris** [SpaceK33z/plex2netflix#27]
Hello, my Plex install has 5 libraries. 4 of these are recognized and work through plex2netflix. One, my 'movies', does not. 
The following error is reported:
```-------
Searching in Movies.
-------
TypeError: Cannot read property 'title' of undefined

**Comment by zobe123** [comment]
same here but i can do a scan if i add the number 999 at the end of the command - but you will get a lot of failed request (code: 504) errors

**Comment by seanvree** [comment]
Same here:

```
c:\plex2netflix>cd c:\plex2netflix && plex2netflix --host 192.168.1.20 --token=DTQyDyypzXkxgnGbvSj7 --country=us --section="TV Shows"
Successfully connected to Plex.
-------
Searching in TV Shows.
-------
TypeError: Cannot read property 'title' of undefined
```

**Comment by danhogwood** [comment]
ditto

**Comment by SpaceK33z** [comment]
Fix is released in `plex2netflix@0.5.4`, thanks for reporting.
